### PR TITLE
test(rxLocalStorage): Add a page object for rxLocalStorage

### PR DESCRIPTION
**Pull request by tomreece**
* [x] Dev LGTM
* [x] Dev LGTM

Added a rxLocalStorage page object with basic functionality to be used by tests in other projects. This saves us from having to write driver.executeScript() all over the place.

### CODE REVIEW

File: src/rxLocalStorage/docs/rxLocalStorage.midway.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    it('should remove something that has been set', function () {
+        rxLocalStorage.setItem('something', 42);
+        rxLocalStorage.removeItem('something');
+        expect(rxLocalStorage.exists('something')).to.eventually.be.false;
+    });
+
+    it('should return true if key exists', function () {
+        rxLocalStorage.setItem('cortana', 123);
+        expect(rxLocalStorage.exists('cortana')).to.eventually.be.true;
+    });
```
Comment (Droogans) [inline src/rxLocalStorage/docs/rxLocalStorage.midway.js]: Please test this if something exists `to.eventually.be.false`.

Comment (tomreece) [inline src/rxLocalStorage/docs/rxLocalStorage.midway.js]: I do that on line 69.

Comment (CITguy) [inline src/rxLocalStorage/docs/rxLocalStorage.midway.js]: Technically, line 69 is testing the behaviour of adding and removing a value.  Whereas, I believe @Droogans wants to test if a value never existed in the first place. Something like...

```javascript
it('should return false if key does not exist', function () {
    // 'siri' hasn't been set
    expect(rxLocalStorage.exists('siri')).to.eventually.be.false
});
```

Am I correct in this assumption?

Comment (tomreece) [inline src/rxLocalStorage/docs/rxLocalStorage.midway.js]: Gotcha. I added a test for that scenario. PR updated.

**Comment by Droogans** [comment]
LGTM && @CITguy please merge.

**Comment by CITguy** [comment]
LGTM

**Pull request merged**